pages/api/awards/collect.js

```diff
@@ -0,0 +1,50 @@
+import prisma from '@prisma/index'; // Prisma client
+import { getSession } from 'next-auth/client'; // Session handling
+
+// Get awards
+export default async (req, res) => {
+  // Collect session from request
+  const session = await getSession({ req });
+
+  //TODO: add functionality of getting awards by performance id
```

Comment:
Is this a use case we need

Reply:
We should get awards by performance for feedback sharing.

Reply:
This is already being done in feedback sharing :)

Reply:
sounds good then we don't need this functionality for now.